tests/k8s/1-038_validate_controller_extra_command_args/02-add-extraCommandArgs.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: example-argocd
+  namespace: test-1-38-controller-extra-command
+spec:
+  controller:
+    extraCommandArgs:
+    - --app-hard-resync
```

Comment:
Can we update the test to verify addition and removal of `extraCommandArgs`? Maybe move this to `01-basic-argocd-install.yaml` i.e create ArgoCD with `extraCommandArgs` and later remove it here.